src/styles/style.css

```diff
@@ -0,0 +1,22 @@
+@import url(Blocks/card.css);
+@import url(Blocks/stars.css);
+
+.page {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: #060b35;

```

Comment:
so if you declared other colors as  variables, maybe it worth to move this color into a variable as well?

Reply:
I use this color as the main color for the page, and in places where it differs - I change. Is it wrong?